.github/workflows/build_next.yml

```
@@ -0,0 +1,30 @@
+name: Build Next.js app
+
+on:
+  pull_request:
```

この提案を取り込めばすぐに修正されます。

```suggestion
on:
  push:
    branches: ["main", "dev"]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: ["main", "dev"]
```

---

`Commit suggestion`

---

後で修正します！

---

@illionillion 
ご指摘の点を修正しました。